Cargo.toml

```diff
@@ -8,9 +8,10 @@ homepage = "https://github.com/mitsuhiko/proguard"
 documentation = "https://docs.rs/proguard"
 license = "BSD-3-Clause"
 readme = "README.md"
+edition = "2018"
```

Comment:
can you temporarily remove this, then run `cargo fix --edition` and then add it back? Now the code is a mix of 2015 and 2018.

Reply:
If I remove the code won't compile. I could re-write it to do so but the parsing code as it's written on `insta` won't build